docs/import-data/overview.mdx

```diff
@@ -147,46 +147,13 @@ export default function ConnectToMemgraph() {
         </section>
         <section>
           <h2 id="kafka">
-            How to import streaming data from Kafka into Memgraph?
+            How to import streaming data into Memgraph?
           </h2>
           <p>
             Memgraph comes with full streaming support, and you can connect
-            Memgraph directly to a Kafka stream. Check out{" "}
-            <a href={useBaseUrl("/memgraph/import-data/kafka")}>this guide</a>{" "}
-            to find out how to connect to a Kafka stream and take a look at the
-            examples with different serialization mechanisms:
+            directly to a Kafka, Redpanda or Pulsar stream {" "}<a href={useBaseUrl("/import-data/data-streams/manage-streams.md")}>using Cypher queries</a>{" "} 
+            or {" "}<a href={useBaseUrl("/import-data/data-streams/manage-streams-lab.md")}>using Cypher queries</a>{" "} 
```

Comment:
I think this is generating build errors.

Reply:
nevermind that now :D that's a whole other world to think about... :) but thnx for telling me

Reply:
OK, i believe i cannot resolve this

Because the page is HTML or whatever it is, I don't understand it, it needs to have /memgraph/something links... but as far as I can understand it, when you have /memgraph/something links, you are linking to the CURRENT version, but these files do not exist in the current version, they are new and exist only in the NEXT version... 

So this is Catch 22... I guess the solution is to have a broken link or no link until we publish the next version of docs... or revamp the import_data/overview page to be MD